### ISSUE TITLE
tf.train 의 정체

### ISSUE THREAD

**Issue by gentian0185** [Com0L/tfStudy#6]
https://www.tensorflow.org/api_docs/python/tf/train#modules

Optimizers
Gradient Computation
Moving Averages
Coordinator and QueueRunner
Reading Summaries from Event Files
Training Hooks
Training Utilities
